spec/requests/api/v1/menu_items_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::MenuItemsController, type: :request do
+    let!(:food1) { create(:menu_item, name: 'Pickled Egg', description: 'An egg that is in a reeeeal pickle!', price: 69) }
```

Comment:
you dont test any of the attributes for these menu items. 

please refactor it to

```
let!(:menu_items) { 3.times { create(:menu_item) } }
```